docs/UserGuide.adoc

```diff
@@ -176,10 +176,20 @@ Example:
 
 * `java seedu.addressbook.Main mydata.txt`
 
+== ENHANCEMENT
```

Comment:
Don't have to add this if you've mentioned it above in the `find` command description (: